datadog_checks_base/datadog_checks/base/utils/db/utils.py

```diff
@@ -271,3 +271,33 @@ def _run_job_rate_limited(self):
 
     def run_job(self):
         raise NotImplementedError()
+
+
+class DatabaseSetting:
```

Comment:
Do we need this class? Could we make do with a `pg_settings` dictionary where the keys match the names in the `pg_settings` table? 

That could be simpler.